src/aria.css

```diff
@@ -116,3 +116,63 @@
     width: fit-content;
     text-align: center;
 }
+
+label:has([role=switch]:last-child) {
+  display: flex;
+  gap: var(--gap);
+  justify-content: space-between;
+}
+
+label[for]:has(+ [role=switch]),
+label[for] + [role=switch],
+[role=switch]:has(+ label[for]),
+[role=switch] + label[for] {
```

Comment:
```suggestion
label[for]:has(+ &),
label[for] + &,
&:has(+ label[for]),
& + label[for] {
```

> I don't particularly like that my solution involves specifying label selectors in the aria.css file, where all the other parent selectors are just [role=thing]. It would be nice if I could move them inside the [role=switch] block somehow (or not include them).